design/buttons.md

```diff
@@ -0,0 +1,14 @@
+## Best Practices
+<!-- docs: start best practices -->
+<!-- docs: start dos -->
+* Use buttons to cause an action or launch a workflow
+* Keep button text short - see "Writing" guidelines
+<!-- docs: end dos -->
+
+<!-- docs: start donts -->
+* Don't use multiple Primary buttons - the primary action should always be clear and obvious
+* Avoid icon-only buttons with unfamiliar icons, only use these for familiar concepts
+* Don't use buttons for navigation, use a link instead
+* Don't open menus with buttons - use a dropdown instead
+<!-- docs: end donts -->
+<!-- docs: end best practices -->
```

Comment:
Remind me again why we wanted the design tips stuff in a separate file vs. just inline? Seems like having this stuff visible in the repo is a nice feature of having it inline. Was it just so that we could set up separate designer owners?

Reply:
It was, yeah. Then they can review any changes made and also only have to really be concerned with these files vs. a situation like always being pulled into changes on the readme.

Reply:
Ok. I'm not sure if the pros outweigh the cons here then? The extra infrastructure to piece these back together again, the fact that these design tips won't be visible in the actual repo documentation vs. how often do we realistically expect this stuff to change and not only that but have a designer notice (after months or years of getting no notifications on it) and have their review catch a problem.

Reply:
Agreed - especially since these are relatively small pieces I think it would be fine to keep them in the same file.